### PR TITLE
add styles to template 💅

### PR DESCRIPTION
**Pull request by jurekbarth**

### CODE REVIEW

File: _templates/component/with-prompt/template.ejs.t [hunk 1]
```diff
@@ -6,5 +6,8 @@ interface <%= h.inflection.transform(name, ['underscore', 'camelize']) %>Templat
 }
 
 export default (render: Function, data: <%= h.inflection.transform(name, ['underscore', 'camelize']) %>TemplateData) => {
-    return render`Fill me`;
+    return render`
+        <style>${require('./style').toString()}</style>
+        <div>Fill me</div>
```
Comment (timomayer) [inline _templates/component/with-prompt/template.ejs.t]: can we have the <div> using the external template file as we discussed it here: https://github.com/biotope/experimental-biotope-components/issues/20

**Comment by jurekbarth** [comment]
@SheepFromHeaven can you explain further?

**Comment by SheepFromHeaven** [comment]
Sorry, copy paste error, everything is fine 😄

File: _templates/component/with-prompt/template.ejs.t [hunk 2]
```diff
@@ -6,5 +6,8 @@ interface <%= h.inflection.transform(name, ['underscore', 'camelize']) %>Templat
 }
 
 export default (render: Function, data: <%= h.inflection.transform(name, ['underscore', 'camelize']) %>TemplateData) => {
-    return render`Fill me`;
+    return render`
+        <style>${require('./style').toString()}</style>
```
Comment (SheepFromHeaven) [inline _templates/component/with-prompt/template.ejs.t]: The file is called index.scss

Comment (SheepFromHeaven) [inline _templates/component/with-prompt/template.ejs.t]:
```suggestion
        <style>${require('./fart').toString()}</style>
```

Comment (SheepFromHeaven) [inline _templates/component/with-prompt/template.ejs.t]:
```suggestion
        <style>${require('./fart').toString()}</style>
```

**Comment by jurekbarth** [comment]
@SheepFromHeaven please accept 👏

**Pull request closed without merging**